examples/anonymous-records/tests/anonymous_records_test.bal

```diff
@@ -9,14 +8,14 @@ int count = 0;
     functionName: "println"
 }
 public function mockPrint(any... s) {
-    outputs[count] = string.convert(s[0]);
+    outputs[count] = s[0].toString();
```

Comment:
Shall we create an issue to improve these tests without using `.toString()`, if we haven't already?

Reply:
Please also mention replacing `outputs[count] = x` with `outputs.push(x)`